### DISCUSSION TITLE
let's vote out the impostor

### DISCUSSION THREAD

**Discussion by JackyBorderCollie** [CluckCluckChicken/Magnifier#38] [Thyroxine-binding globulin]
in this game you need to guess who is the impostor and vote him if it's right yOu gEt aMoGuS dRiP fOr fReE!1!!!!!!!11!!!1!! but if you don't. YOU GET EJECTED FROM DA SHIP!!!!!!!!!!!!

also when u correctly guess next round beggings

LES STARTTTTTTTTTTT!!!!!!!!!!!!!!!!!!!!!!!!!

also there are the colors:
red
yellow
orange
green (unused oMg!1!!!!1111!!!)
blue
lime
light blue
black
white

also u can post clues

i start:

the sussy imposter had a color between red and lime

**Comment by JackyBorderCollie** [comment]
come on

**Comment by CubeyTheCube** [comment]
unfunny, close this

**Reply by JackyBorderCollie** [reply]
no it's a game

**Comment by JackyBorderCollie** [comment]
sussy bump
